### PR TITLE
Add Schemathesis to IMPLEMENTATIONS.md

### PR DESCRIPTION
**Pull request by Stranger6667**
As a replacement for #2206

### CODE REVIEW

File: IMPLEMENTATIONS.md [hunk 1]
```diff
@@ -52,10 +52,11 @@ These tools are not endorsed by the OAI.
 | angular-swagger-ui | [github/angular-swagger-ui](https://github.com/Orange-OpenSource/angular-swagger-ui) | AngularJS | An angularJS implementation of Swagger UI |
 | Redoc | [github/Redocly/redoc](https://github.com/Redocly/redoc) | JavaScript | A React-based renderer with deep support for OAS v2 and v3 and zero dev-dependency|
 
-#### Mock Servers
+#### Testing tools
```
Comment (hkosova) [inline IMPLEMENTATIONS.md:55]: Please create a separate section for Testing Tools. Testing is different from mocking.

File: IMPLEMENTATIONS.md [hunk 2]
```diff
@@ -52,10 +52,11 @@ These tools are not endorsed by the OAI.
...
 | Title          | Project Link | Language | Description |
 | -------------- | ------------ | -------- | ----------- |
 | API Sprout     | [github/danielgtaylor/apisprout](https://github.com/danielgtaylor/apisprout) | Go | Lightweight, blazing fast, cross-platform OpenAPI 3 mock server with validation |
+| Schemathesis   | [github/schemathesis/schemathesis](https://github.com/schemathesis/schemathesis) | Python | A modern API testing tool for web applications built with Open API and GraphQL specifications |
```
Comment (hkosova) [inline IMPLEMENTATIONS.md:59]: Please change "Open API" to OpenAPI (no space).

**Review by hkosova — CHANGES_REQUESTED** [review]

File: IMPLEMENTATIONS.md (hunk 1 above)
Comment (Stranger6667) [reply IMPLEMENTATIONS.md]: Done!

File: IMPLEMENTATIONS.md (hunk 2 above)
Comment (Stranger6667) [reply IMPLEMENTATIONS.md]: Done!

**Review by freelancing-solutions — COMMENTED** [review]
https://appdev.consulting.redhat.com/tracks/contract-first/automated-testing-with-schemathesis.html

**Review by MikeRalphson — APPROVED** [review]
LGTM

**Review by darrelmiller — APPROVED** [review]

**Pull request merged**